src/issue/screens/issue.screen.js

```diff
@@ -78,11 +79,22 @@ class Issue extends Component {
                 issue: state.params.issue,
               })}
           />
-        ),
-      };
-    }
+        );
+      }
+
+      return (
+        <Icon
```

Comment:
It seems that when press it, there should be a change in the transparency of the icon, but it's not now, can we add?

Reply:
Hey @lex111, I'm using this icon like have in other places. I think all the right icons in the app has `underlayColor` transparent. That being said, I think if we would like to add this, we should handle in another PR. What do you think about? 😄

Reply:
@jouderianjr ok, it's just weird that it's not done now for README and issue screens.